examples/simple_example.py

```diff
@@ -18,18 +20,16 @@ def get_current_weather(location, unit="fahrenheit"):
 
 
 def run_conversation():
-    openai.api_key = os.getenv("OPENAI_API_KEY")
-
     messages = [{"role": "user", "content": "What's the weather like in London?"}]
 
-    response = openai.ChatCompletion.create(
+    response = client.chat.completions.create(
```

Comment:
Is this still backwards compatible

Reply:
No it is not, so I decided to update the version from 1.0.x to 1.1.x